### ISSUE TITLE
CUDA easyblock must protect usage of $HOME

### ISSUE THREAD

**Issue by akesandgren** [easybuilders/easybuild-easyblocks#1682]
During installation of CUDA there are accesses to $HOME:
sh: 1: cannot create /home/a/ake/.local/share/applications/nsight.desktop: Permission denied

sh: 1: cannot create /home/a/ake/.local/share/applications/nvvp.desktop: Permission denied

We need to handle this.

**Comment by boegel** [comment]
Agreed.

One thing we could try is to redefine `$HOME` while installing CUDA to the temporary directory available during each EasyBuild session?

**Comment by akesandgren** [comment]
Yeah, that as my idea. I just couldn't remember what to do so I left it out of https://github.com/easybuilders/easybuild-easyblocks/pull/1684
